pt-BR/getting_started.md

```diff
@@ -1772,18 +1772,18 @@ right places.
 
 ![Article with Comments](images/getting_started/article_with_comments.png)
 
-Refactoring
+Refatorando
 -----------
 
-Now that we have articles and comments working, take a look at the
-`app/views/articles/show.html.erb` template. It is getting long and awkward. We
-can use partials to clean it up.
+Agora que nossos artigos e comentários funcionam, dê uma olhada no template
+`app/views/articles/show.html.erb`. Ele está ficando longo e esquisito. Nós
+podemos usar partials (parciais) para melhorá-lo.
```

Comment:
```suggestion
podemos usar *partials* (parciais) para melhorá-lo.
```
Algumas palavras estamos usando sem traduzir e com itálico. Se quiser dar uma olhadinha, subimos um Manual de Tradução essa semana - https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md

Reply:
Talvez aqui, por ser uma introdução, entre parenteses poderíamos descrever um pouco melhor as _partials_ como _views parciais_ por exemplo.